lib/idp/constants.rb

```diff
@@ -127,5 +127,10 @@ module Vendors
     MOCK_IDV_APPLICANT_FULL_STATE_ID_JURISDICTION = 'North Dakota'
     MOCK_IDV_APPLICANT_FULL_STATE = 'Montana'
     MOCK_IDV_APPLICANT_FULL_IDENTITY_DOC_ADDRESS_STATE = 'Virginia'
+
+    ALLOWED_IALMAX_PROVIDERS = %w[
```

Comment:
is this the right place for this allowlist? i wasn't sure if there were other places it might go/naming conventions i was unaware of

Reply:
It should probably be in a config so that we don't have to manage it in the application code.

Reply:
yay thanks!!!  fixed in [`514fb7d` (#8700)](https://github.com/18F/identity-idp/pull/8700/commits/514fb7d9e48cf667422917ad12cbb42a3a39bc48)